### ISSUE TITLE
Types of policies that can be evaluated

### ISSUE THREAD

**Issue by AndreaCimminoArriaga** [w3c/odrl#70]
The [formal semantics document](https://w3c.github.io/odrl/formal-semantics/#:~:text=A%20odrl%3ASet%20policy%20must%20be%20considered%20by%20an%20ODRL%20Evaluator.%20The%20odrl%3ASet%20is%20not%20necessarily%20linked%20to%20odrl%3AOffer%20nor%20odrl%3AAgreement.) specifies that an evaluator should consider only `odrl:Set`. Nevertheless, specifying this would entail a retrocompatibility to any policy defined until now with the ODRL semantics SHOULD be evaluable. I think, IMHO, it would be interesting to have a sub-type, something like odrl:EnforceablePolicy or odrl:EvaluableSet, etc. to mark the policies that MUST be enforceable from now on if they use such type.

**Comment by joshcornejo** [comment]
I don't agree with `odrl:Set` being the only one to be evaluated, IMHO it is the least likely candidate.

At its simplest, evaluations are "can party **Adam**, perform Action **Use** over asset **Car**", without a party in the `odrl:assignee` role, you are not evaluating a complete policy.

The only tangible/actionable policies (those that have actions that are performed by assignees), are of the class `odrl:Agreement`.

An odrl:Set doesn't need to make sense (read "**any**" below):

- `odrl:Set` =  "An ODRL Policy of subclass [Set](https://www.w3.org/TR/odrl-model/#policy-set)(An ODRL Policy of subclass Agreement represents Rules that have been granted from assigner to assignee Parties.) represents **any** combination of Rules." A placeholder for rules, sort of catalogue;
- `odrl:Offer` = "An ODRL Policy of subclass [Offer](https://www.w3.org/TR/odrl-model/#policy-offer) represents Rules that are being offered from assigner Parties.";
- `odrl:Request` = a request to match an `odrl:Offer`
- `odrl:Agreement` = "An ODRL Policy of subclass [Agreement](https://www.w3.org/TR/odrl-model/#policy-agreement) represents Rules that have been granted from assigner to assignee Parties"

**Comment by AndreaCimminoArriaga** [comment]
I missed to add also `odrl:Agreement` as evaluable policy. I just edited the issue
 
`I don't agree with odrl:Set being the only one to be evaluated, IMHO it is the least likely candidate.` 
This is written in the formal semantics document, I'm just citing it. 

In any case, I think that your comment is somehow aligned with the fact that maybe a new type to specify policies that an evaluator must be able to understand can be an interesting ad-on to have.

**Comment by fornaran** [comment]
A policy with type "Agreement" MUST have one assignee property value, and is considered by an ODRL Evaluator.
A policy with type "Set" is used to formalize those policies where it makes sense to not specify the assignee and it  is considered by an ODRL Evaluator.

**Comment by fornaran** [comment]
Following the ODRL Core Vocabulary https://www.w3.org/TR/odrl-vocab/#term-Set:
"A Set Policy MUST contain a target Asset, and at least one Rule. A Set Policy is the default Policy subclass. The Set is aimed at scenarios where there is an open criteria for the semantics of the policy expressions and typically refined by other systems/profiles that process the information at a later time. **No privileges are granted to any Party (if defined).**"
Should we remove the examples of Set Policies from the Semantics of ODRL page?
